db/migrate/20221109120726_add_epoch_pythnet_to_sol_price.rb

```diff
@@ -0,0 +1,5 @@
+class AddEpochPythnetToSolPrice < ActiveRecord::Migration[6.1]
+  def change
+    add_column :sol_prices, :epoch_pythnet, :integer
```

Comment:
We agreed that we don't need to add `epoch_pythnet` to the `sol_prices` table as it is not used in any view

Reply:
Recently, It turned out as above.